test/issue224.jl

```diff
@@ -0,0 +1,20 @@
+"""
+Regression test for https://github.com/JuliaPolyhedra/Polyhedra.jl/issues/224
+"""
+function issue224test(lib::Polyhedra.Library)
+    p = polyhedron(convexhull([-5012508711568619//562949953421312, 583069911833861//562949953421312, 1219842180319805//35184372088832],
```

Comment:
Would an example with a lower dimensional  polyhedron allow to reproduce the issue ? E.g. with the simplex in `test/simplex.jl` which gives a 2 dimensional object in 3D.

Reply:
It's not obvious to me how to reproduce this error on hand-crafted data.  The example here, found in the wild, has the following properties:
* `dim(p) == fulldim(p) == 3`
* `volume(p)` gives a `DimensionMismatch` error, due to a simplex having too few vertices due to the numerics issues highlighted in this PR
* The issue persists even after calling `removevredundancy!`, `removehredundancy!`, or both.

I tried it with `p = convexhull([0, 0, 0], [1, 0, 0], [0, 1, 0], [0, 0, 1 + ε])` for small `ε`, and similar, but the error didn't show up in those cases.